client-app/pages/account/loyalty.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="bg-gray-100 flex-grow pt-6 pb-16 shadow-inner">
+    <div class="max-w-screen-2xl md:px-12 mx-auto">
+      <div class="flex lg:space-x-5">
+        <!-- First column-->
+        <div class="hidden lg:flex flex-col lg:w-1/5 space-y-5">
+          <AccountNavigation></AccountNavigation>
+        </div>
+
+        <!-- Second column-->
+        <div class="flex flex-col w-full lg:w-4/5 space-y-5">
+          <div class="flex justify-between items-center mx-5 lg:mx-0">
+            <h2 class="text-gray-800 text-3xl font-bold uppercase">Loyalty</h2>
+          </div>
+          <div class="bg-white shadow-sm rounded border px-7 py-7 md:px-9 md:py-8">
+            <div class="flex justify-between items-center">
+              <div class="text-gray-800 text-3xl font-bold">User Balance: {{ userBalance?.amount }}</div>
+              <VcButton class="float-right uppercase px-5" @click="addOperation"> Add operation </VcButton>
+            </div>
+          </div>
+          <div class="bg-white shadow-sm rounded border px-7 py-7 md:px-9 md:py-8">
+            <div class="flex justify-between items-center pb-5">
+              <div class="flex flex-col gap-5">
+                <VcRadioButton id="pagination" v-model="scrollMode" value="pagination" :label="'Pagination view'" />
+                <VcRadioButton id="scroll" v-model="scrollMode" value="scroll" :label="'Infinity scroll view'" />
+              </div>
+              <div>
+                <VcSelect
+                  v-model="itemsPerPageStr"
+                  :items="_.range(5, 26)"
+                  :label="'Items per page'"
+                  :is-required="true"
```

Comment:
Plain boolean props can be written just like `is-required`. Not needed to pass `true` explicitly.